database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -113,18 +132,21 @@ trait MetadataSlickDatabase extends MetadataSqlDatabase {
   }
 
   override def refreshMetadataSummaryEntries(metadataKey1: String, metadataKey2: String, metadataKey3: String,
-                                             metadataKey4: String, buildUpdatedSummary:
+                                             metadataKey4: String, metadataLabelKey: String, buildUpdatedSummary:
```

Comment:
They used to have better names and were changed to this for reasons I don't remember (not by me)